src/Analysis/GWASV2/Shared/wizardEndpoints/cohortMiddlewareApi.js

```diff
@@ -60,6 +60,34 @@ export const fetchOverlapInfo = async (
   return getOverlapStats.json();
 };
 
+// Basically a copy of fetchOverlapInfo above, but without the HARE arguments:
```

Comment:
I could go either way on this one and am fine with whatever you decide:

I was thinking the code could be made more DRY by having a conditional in fetchOverlapInfo() for this case, but I think a little bit of repeated code will probably be easier to understand and lead to a simpler function. However it might be trickier to maintain and results in duplicate code.

Reply:
I'll leave as is for now. The comment line above might help in a future refactoring/cleanup